constants.ts

```diff
@@ -0,0 +1 @@
+export const APICALL = false;
```

Comment:
Hey Pooria,

Just a quick suggestion:


1. Consider changing the flag name to something else for better clarity. Ensure it follows naming conventions ([Coding Style Guide](https://github.com/OpenBCca/openbc-web/wiki/Coding-Style-Guide#naming)). (e.i. `UseGithubAPI`)
2. As Sam suggested, it is better to move the constants file to the Next.js configuration file because the constants file is not part of our convention.
3. Please add unit tests for the changes made.

Reply:
I also vouch for these suggestions, i think making the flag a `process.env` is more suitable than a `const`